### DISCUSSION TITLE
Incompatible version of Windows for OS 2019

### DISCUSSION THREAD

**Discussion by Moogucat** [MicrosoftDocs/WDAC-Toolkit#167] [Q&A]
![image](https://user-images.githubusercontent.com/100182169/203569204-3d9e4ee7-8523-4512-9048-994714e053a2.png)
Hello 
i have installed the latest version of WDAC wizard in OS 2019 version, everytime i open the APP, it pops out this error message.

**Comment by Moogucat** [comment]
Will this cause any problem?

**Reply by jgeurten** [reply]
It likely will cause problems creating the policies. While the version is supported (OS 2019), this message is shown on devices running Windows Home SKU which does not have the PS module in order to create and merge policies.
